src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
 // write your code here
+const rows = document.querySelectorAll('table tr');
+
+for (const i of rows) {
```

Comment:
The variable name `i` is not descriptive. Consider using a more meaningful name like `row` to improve code readability.